@xen-orchestra/web-core/lib/components/ui/alarm-item/UiAlarmItem.vue

```diff
@@ -0,0 +1,144 @@
+<!-- v1 -->
+<template>
+  <li class="ui-alarm-item" :class="className">
+    <div class="content">
+      <div class="label-value text-ellipsis">
+        <UiButtonIcon
+          v-if="alarm.description"
+          :icon="isDescriptionVisible ? faAngleDown : faAngleRight"
+          size="small"
+          accent="brand"
+          @click="toggleDescription()"
+        />
+        <span class="typo-body-regular text-ellipsis">
+          {{ alarm.label }}
+        </span>
+        <span class="typo-body-regular value">
+          {{ alarm.value }}
+        </span>
+      </div>
+      <div class="typo-body-regular-small info">
+        <div v-if="slots['object-link']" class="object-link">
+          <span class="on-text">
+            {{ $t('on-object') }}
+          </span>
+          <span class="descriptor">
+            <slot name="object-link" />
+          </span>
+          <span class="divider" />
+        </div>
+        <span class="typo-body-regular-small">
+          {{ timeAgo }}
+        </span>
+      </div>
+    </div>
+    <div v-if="isDescriptionVisible" class="description">
+      {{ alarm.description }}
+    </div>
+  </li>
+</template>
+
+<script lang="ts" setup>
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import { toVariants } from '@core/utils/to-variants.util'
+import { faAngleDown, faAngleRight } from '@fortawesome/free-solid-svg-icons'
+import { useTimeAgo, useToggle } from '@vueuse/core'
+import { computed } from 'vue'
+
+type Alarm = {
+  label: string
+  value: string
+  date: Date | number | string
+  description?: string
+}
+
+const { alarm, size } = defineProps<{
+  alarm: Alarm
+  size: 'small' | 'large'
+}>()
+
+const slots = defineSlots<{
+  'object-link'?(): any
+}>()
+
+const timeAgo = computed(() => useTimeAgo(alarm.date))
```

Comment:
`useTimeAgo` is reactive so the _computed_ is not needed.
But as we destructure props, to keep reactivity on the date, you will have to use a getter:
```suggestion
const timeAgo = useTimeAgo(() => alarm.date)
```
The other thing here, is that `useTimeAgo` does not handle localization by default.
We should make a helper to handle this, I will create a card in Plane, so you don't have to worry about it in this PR.